packaging/installer/UPDATE.md

```diff
@@ -64,9 +64,21 @@ that prefix to this command to make sure it finds Netdata.
 wget -O /tmp/netdata-kickstart.sh https://my-netdata.io/kickstart.sh && sh /tmp/netdata-kickstart.sh
 ```
 
-> ❗ If the above command fails, you can [reinstall
-> Netdata](/packaging/installer/REINSTALL.md#one-line-installer-script-kickstartsh) to get the latest version. This also
-> preserves your [configuration](/docs/configure/nodes.md) in `netdata.conf` or other files.
+### For older binpkg installs
+
+The above command is known not to work with binpkg type installs for stable releases with a version number of
+v1.33.1 or earlier, and nightly builds with a version number o v1.33.1-93 or earlier. If you have such a system,
+the above command will report that it found an existing install, and then issue a warning about not being able to
+find the updater script.
+
+On such installs, you can update Netdata using your distribution package manager.
+
+### If that does not work
```

Comment:
Just as a clarification: This section generally specifies what to do if the kickstart script doesn't work, right?

If so, I would place the section right after the install code block. 

If the information specifies resolving issues with binpkg installs then I suggest not to use a new heading but just to make it bold.

Reply:
Yes, it’s for the generic case of the script not working.

Personal experience has taught me that when listing solutions for failure conditions in documentation like this the most general case should be listed last, otherwise a nontrivial number of people will ignore the more specific (and more correct) cases that apply to them and just go with whatever the most generic resolution is.

I agree though that the heading name should probably be different. Perhaps ‘If the kickstart script does not work’?